docs/usage-as-end-user.md

```diff
@@ -0,0 +1,27 @@
+# Using the Ubuntu extension with Gardener as end-user
+
+The [`core.gardener.cloud/v1beta1.Shoot` resource](https://github.com/gardener/gardener/blob/master/example/90-shoot.yaml) declares a few fields that must be considered when this OS extension is used.
+
+In this document we describe how this configuration looks like and under which circumstances your attention may be required.
+
+## AWS VPC settings for Ubuntu workers
+
+Gardener allows you to create Ubuntu based worker nodes by:
+1. Using a Gardener managed VPC
+2. Reusing a VPC that already exists (see example below)
+   ```yaml
```

Comment:
I would not copy this here but rather put a link to the AWS extension's docu - prevent's double maintenance in case of changes.